### PR TITLE
changes for sepsis metadata and tags and UI changes

### PR DESCRIPTION
**Pull request by krupalip**
changes for sepsis metadata and tags
- [Sepsis: 2d libraries - raw data - Changes to metadata and tags ](https://github.com/BioplatformsAustralia/bpa-archive-ops/issues/708)
- [Sepsis: Protein database dataset- Changes to tags](https://github.com/BioplatformsAustralia/bpa-archive-ops/issues/709)
- [Sepsis: Sepsis - Update file description (analysed data)](https://github.com/BioplatformsAustralia/bpa-archive-ops/issues/710)

UI:
- [Sepsis and general: Align the following text to the centre of the page](https://github.com/BioplatformsAustralia/bpa-archive-ops/issues/711)

### CODE REVIEW

File: ckanext/bpatheme/templates/snippets/bpa_register.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 {% if not c.userobj %}
 <div class="row">
 <div class="span2"></div>
-<div class="span5" style="font-size: 130%; text-align: center;">
+<div class="span" style="font-size: 130%; text-align: center;">
```
Comment (grahame) [inline ckanext/bpatheme/templates/snippets/bpa_register.html]: is this meant to be in here? it looks like a mistake?

Comment (krupalip) [reply ckanext/bpatheme/templates/snippets/bpa_register.html]: Its just a small change so I thought to push to all together.

Comment (krupalip) [reply ckanext/bpatheme/templates/snippets/bpa_register.html]: @grahame  is this ok?

Comment (grahame) [reply ckanext/bpatheme/templates/snippets/bpa_register.html]: I'm pretty sure 'span' isn't a defined CSS class; 'span5' is a bootstrap 2 class that makes an element span 5 columns... why are you making this change?

**Pull request merged**